templates/zerver/compose.html

```diff
@@ -105,8 +105,7 @@
                   <a class="drafts-link" href="#drafts" title="{{ _('Drafts') }} (d)">{{ _('Drafts') }}</a>
                   <span id="sending-indicator">{{ _('Sending...') }}</span>
                   <div id="send_controls" class="new-style">
-                    <input type="checkbox" id="enter_sends" name="enter_sends" value="enter_sends" />
-                    <label id="enter-sends-label" class="compose_checkbox_label" for="enter_sends">{{ _('Press Enter to send') }}&nbsp;</label>
+                    <label id="enter-sends-label" class="compose_checkbox_label"><input type="checkbox" id="enter_sends" />{{ _('Press Enter to send') }}</label>
```

Comment:
Having the input and label as separate lines makes it easier to deal with in e.g. the chrome developer console.

Reply:
@rishig Split into two separate lines 👍